linux/kernel/updating.md

```diff
@@ -8,7 +8,7 @@ If you use the standard Raspbian update/upgrade process (found [here](../../rasp
 
 To use rpi-update, execute it using sudo as follows:
 
-```sudo rpi-update
+```sudo rpi-update```
```

Comment:
Thanks. I think the 3 backticks (both before and after) should all be on a separate line though.

Reply:
It's still a valid MD syntax, but for the sake of readability, I've put the backticks on separate lines.